src/components/incompatibleSchemaChange/Form.tsx

```diff
@@ -87,7 +88,7 @@ export default function IncompatibleSchemaChangeForm({
             ) : null}
 
             {/* The presence of ListboxComponent in autoCompleteDefaultProps prevents
-                the options from rendering  */}
+                the options from rendering. */}
```

Comment:
I think this comment needs replaced - this is now just a TODO to get this `Autocomplete` using the defaults since that `Listbox...` issue was handled in another PR.